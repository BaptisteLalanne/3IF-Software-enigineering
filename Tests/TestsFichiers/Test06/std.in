1
1.0
43.9
-1.1
2019-01-01
2019-01-04
0