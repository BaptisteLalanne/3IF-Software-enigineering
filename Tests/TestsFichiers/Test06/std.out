Veuillez-renseigner les informations suivantes:
Latitude :
Longitude :
Rayon :
Date de début :
Date de fin :
Attention, la zone sélectionnée n'est couverte par les capteurs qu'à


rayonMesureCapteur=400
tailleCarte=2*1*10^3=2000
centreCapteurLongitude:(-1-(-1,1-1))*10^3=1100
centreCapteurLatitude : (44-(43,9-1))*10^3=1100
debutMesureCapteurLongitude = 1100-400=700
finMesureCapteurLongitude= 1100+400=1500
debutMesureCapteurLatitude=centreCapteurLatitude-rayonMesureCapteur=1100-400=700
finMesureCapteurLatitude=centreCapteurLatitude+rayonMesureCapteur=1500
stop à la ligne 202